src/credssp/sspi_cred_ssp/mod.rs

```diff
@@ -1,3 +1,4 @@
+#[cfg(not(target = "wasm32-unknown-unknown"))]
```

Comment:
Likewise, if it’s only for windows maybe `#[cfg(windows)]` is better

Reply:
Yeah. I'm working on it